### ISSUE TITLE
[android] NPE @ com.andrew.apollo.dragdrop.DragSortListView.measureItemAndGetHeights:604

### ISSUE THREAD

**Issue by gubatron** [frostwire/frostwire#176]
```
java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.ViewGroup$LayoutParams android.view.View.getLayoutParams()' on a null object reference
	at com.andrew.apollo.dragdrop.DragSortListView.measureItemAndGetHeights(DragSortListView.java:604)
	at com.andrew.apollo.dragdrop.DragSortListView.getItemHeights(DragSortListView.java:701)
	at com.andrew.apollo.dragdrop.DragSortListView.getShuffleEdge(DragSortListView.java:780)
	at com.andrew.apollo.dragdrop.DragSortListView.updatePositions(DragSortListView.java:828)
	at com.andrew.apollo.dragdrop.DragSortListView.layoutChildren(DragSortListView.java:1445)
```

**Comment by gubatron** [comment]
@votaguz @alejandroarturom ? probably an easy fix, 10mins at most.

**Comment by gubatron** [comment]
Fixed 1a1975a
